spec/command/package_spec.rb

```diff
@@ -73,10 +73,39 @@ module Pod
         command.run
 
         lib = Dir.glob("NikeKit-*/ios/NikeKit.framework/NikeKit").first
-        `lipo #{lib} -verify_arch armv7 armv7s arm64`
+        `lipo #{lib} -verify_arch x86_64 i386 armv7 armv7s arm64`
         $?.success?.should == true
       end
 
+      it "includes Bitcode for device arch slices when packaging an iOS Pod" do
+        SourcesManager.stubs(:search).returns(nil)
+
+        command = Command.parse(%w{ package spec/fixtures/NikeKit.podspec })
+        command.run
+
+        lib = Dir.glob("NikeKit-*/ios/NikeKit.framework/NikeKit").first
+
+        #Check for __LLVM segment in each device architecture
```

Comment:
According to https://forums.developer.apple.com/thread/3971, this is the best way to check for a Bitcode segment.